src/include/block.h

```diff
@@ -205,12 +205,24 @@ struct __wt_bm {
     int (*write)(WT_BM *, WT_SESSION_IMPL *, WT_ITEM *, uint8_t *, size_t *, bool, bool);
     int (*write_size)(WT_BM *, WT_SESSION_IMPL *, size_t *);
 
-    WT_BLOCK *block; /* Underlying file */
+    WT_BLOCK *block; /* Underlying file. */
```

Comment:
Can you include in the comment that this is the active file, as at this point, we can have multiple underlying files?